app/ui/src/app/integration/import-export/import/integration-import.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+import { ApiHttpService } from '@syndesis/ui/platform';
+import { IntegrationImportData, IntegrationImportRequest } from './integration-import.models';
+
+@Injectable()
+export class IntegrationImportService {
+  constructor(private apiHttpService: ApiHttpService) { }
+
+  getImport(id: string): Observable<IntegrationImportData> {
```

Comment:
Don't think we need ti s class, we can't upload then review for imports.  Once the file is summitted to the importer, your done.  It's imported.  You just need to display the import results.

Reply:
I am removing the `fetch`-related Redux actions, hence this being a WIP.